README.md

```diff
@@ -202,7 +202,7 @@ Creating an IPFS instance couldn't be easier, all you have to do is:
 // Create the IPFS node instance
 const node = new IPFS()
 
-node.on('start', () => {
+node.on('ready', () => {
```

Comment:
It is really `start` that you should be looking for. 
https://github.com/ipfs/js-ipfs/issues/819#issuecomment-290169511